deb/debian-buster/Dockerfile.aarch64

```diff
@@ -6,7 +6,8 @@ RUN sed -ri "s/(httpredir|deb).debian.org/$APT_MIRROR/g" /etc/apt/sources.list
 
 RUN apt-get update && apt-get install -y apparmor bash-completion btrfs-tools build-essential cmake curl ca-certificates debhelper dh-apparmor dh-systemd git libapparmor-dev libdevmapper-dev libltdl-dev libseccomp-dev pkg-config vim-common libsystemd-dev gnupg dirmngr --no-install-recommends && rm -rf /var/lib/apt/lists/*
 
-ENV GO_VERSION 1.10.2
+ARG GO_VERSION
+ENV GO_VERSION $GO_VERSION
```

Comment:
Is `GO_VERSION` used when generating the `docker info` / `docker version` output?

I'm ok with keeping it in for now, but we can decide to remove it at some point (if it doesn't add value; i.e. we don't use the actual images, right?)


Was also looking if we should set a default, but I don't think there's a (e.g.) `-latest` download-url, so I guess we can't

Reply:
@thaJeztah No that's populated by the `runtime` package in go:

https://github.com/docker/cli/blob/master/cli/command/system/version.go?utf8=%E2%9C%93#L140-L154

Reply:
Was trying to look into if we could use git tags from the main `golang/go` repository to find the actual "latest" but it doesn't look like that's the best approach either since they have long living release branches which could squash the actual "latest" build.